src/data.table.h

```diff
@@ -78,6 +78,15 @@
 #define NEED2UTF8(s) !(IS_ASCII(s) || (s)==NA_STRING || IS_UTF8(s))
 #define ENC2UTF8(s) (!NEED2UTF8(s) ? (s) : mkCharCE(translateCharUTF8(s), CE_UTF8))
 
+// R has been providing a widely portable definition, but since that's not documented, define our own too
```

Comment:
Indeed I'm surprised it's so uncommon:

https://github.com/search?q=org%3Acran+path%3Asrc+%2F%28%5E%7C%5Cn%7C+%29NORET+%2F&type=code